README.md

```diff
@@ -65,8 +65,9 @@ In the list of commands (<kbd>CTRL</kbd> + <kbd>SHIFT</kbd> + <kbd>P</kbd>), the
 ## Extension Settings
 
 * `glTF.defaultV1Engine` - Choose the default 3D engine that will render a glTF 1.0 model in the preview window.
-
 * `glTF.defaultV2Engine` - Choose the default 3D engine that will render a glTF 2.0 model in the preview window.
+* `glTF.Babylon.environment` - Override the default reflection map for the BabylonJS glTF preview window.
```

Comment:
It's probably worth a bit more documentation here on what type of values are expected here....for Babylon it would be a dds file (and maybe a pointer to how to create a different one like [this](http://doc.babylonjs.com/overviews/physically_based_rendering#creating-a-dds-environment-file-from-an-hdr-image)), and similar information for the ThreeJs environment.  Users of the extension might not be that familiar with the renderers, but may want to use the extension to simply view objects, so anything we can do to reduce the friction of improving their experience, the better.